src/locale/locales/it/messages.po

```diff
@@ -5639,26 +5639,26 @@ msgstr "Rispondi a <0><1/></0>"
 #: src/view/com/posts/FeedItem.tsx:513
 msgctxt "description"
 msgid "Reply to a blocked post"
-msgstr ""
+msgstr "Rispondi ad un post bloccato"
```

Comment:
```suggestion
msgstr "Risposta a un post bloccato"
```
This string is referring to a reply, rather than an action, so DeepL tells me it would be `Risposta`?